notebooks/00_quick_start/run_notebook_on_azureml.ipynb

```diff
@@ -0,0 +1,275 @@
+    "- As with other Azure services, there are limits on certain resources (for eg. AmlCompute quota) associated with the Azure Machine Learning service. Please read [these instructions](https://docs.microsoft.com/en-us/azure/azure-supportability/resource-manager-core-quotas-request) on the default limits and how to request more quota.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 33,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "NOTEBOOK_NAME = 'sar_movielens.ipynb' # use different notebook file name if trying to run other notebooks\n",
+    "experiment_name = NOTEBOOK_NAME.strip(\".ipynb\")\n",
+    "tmpdir = TemporaryDirectory()\n",
+    "notebook_path = path.join(tmpdir.name, NOTEBOOK_NAME)\n",
+    "shutil.copy(NOTEBOOK_NAME, notebook_path)\n",
+    "        \n",
+    "exp = Experiment(workspace=ws, name=experiment_name)\n",
+    "\n",
+    "run_config_aml = RunConfiguration()\n",
+    "run_config_aml.target = \"amlcompute\" # possible to use different compute targets\n",
+    "run_config_aml.amlcompute.vm_size = 'STANDARD_D2_V2'\n",
+    "run_config_aml.environment.docker.enabled = True\n",
+    "run_config_aml.environment.docker.base_image = DEFAULT_CPU_IMAGE\n",
+    "run_config_aml.environment.python.user_managed_dependencies = False\n",
+    "run_config_aml.auto_prepare_environment = True\n",
+    "run_config_aml.environment.python.conda_dependencies = CondaDependencies.create(python_version='3.6.2')\n",
+    "run_config_aml.environment.python.conda_dependencies.add_pip_package('sklearn')\n",
+    "\n",
+    "cfg = NotebookRunConfig(source_directory='../../',\n",
+    "                            notebook='notebooks/00_quick_start/' + NOTEBOOK_NAME,\n",
+    "                            output_notebook='outputs/out.ipynb',\n",
+    "                            parameters={\"MOVIELENS_DATA_SIZE\": \"100k\", \"TOP_K\": 10},\n",
+    "                            run_config=run_config_aml)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`exp.submit()` will submit source_directory folder and designate notebook to run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "run = exp.submit(cfg)\n",
+    "run"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Above cell should output similar table as below\n",
+    "![Experiment submit output](https://recodatasets.blob.core.windows.net/images/aml_papermill_cell_output.jpg)\n",
+    "After clicking \"Link to Azure Portal\", experiment run details tab looks like this\n",
+    "![Azure Portal Experiment](https://recodatasets.blob.core.windows.net/images/aml_papermill_portal.jpg)\n",
+    "Experiment output is out.ipynb. You can download and view this file locally.\n",
```

Comment:
This is not the experiment output. The metrics are logged to the run, and the out.ipynb is the notebook that was initially submitted. I don't think we need the image

Reply:
I just tried submitting a notebook without output and out.ipynb comes with output. I think out.ipynb is the notebook after the run

Reply:
I just meant that the text is incorrect. It is not output from the experiment. They can see the notebook that was submitted to the experiment logged as out.ipynb